LinkedList/MyLinkedList.cpp

```diff
@@ -0,0 +1,98 @@
+#include "MyLinkedList.h"
+#include <corecrt_malloc.h>
+#include <iostream>
+
+Node* head = nullptr;
+
+void MyLinkedList::Push(TYPE newValue)
+{
+	Node* newNode = (Node*) malloc(sizeof(Node));
+	newNode->value = newValue;
+	newNode->pNext = head;
+	head = newNode;
+}
+
+void MyLinkedList::Pop()
+{
+	Node* tempPtr;
+	tempPtr = head->pNext;
+	free(head);
+	head = tempPtr;
+}
+
+void MyLinkedList::Insert(int index, TYPE value)
+{
+	Node* newNode = (Node*)malloc(sizeof(Node));

```

Comment:
It's not that case when u should use "malloc"/"free" instead of "new"/"delete". So, plz, replace all uses of malloc/free with new/delete